client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -78,10 +78,13 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 		}
 	};
 
+	/* translators: Link to localized WordPress.com terms of service if it exists */
+	const localizedTosLink = __( 'https://wordpress.com/tos/' );
```

Comment:
Let's use [`localizeUrl()`](https://github.com/Automattic/wp-calypso/blob/a4616438bcb848e97b784fa6d55ba4e9c5cb3d21/client/lib/i18n-utils/utils.js#L221)

Reply:
That's awesome! I'm curious to see what the bundle impact will be:

https://github.com/Automattic/wp-calypso/blob/a4616438bcb848e97b784fa6d55ba4e9c5cb3d21/client/lib/i18n-utils/utils.js#L5

I wonder if we'd successfully avoided that until now…

Reply:
True! FYI @sgomes in case you've come accross optimizing i18n things previously.

Reply:
Has Gutenboarding really managed to avoid `i18n-calypso` so far? If so, then you're probably going to see a big bump by pulling it in, but to be fair that's not something I've really looked at too much, since it's everywhere in Calypso. I have worked on making `i18n-calypso` smaller and faster, but never on removing/adding it.

Reply:
Current ICFY report:

```
Chunk Groups Delta:
name                 stat_size           parsed_size           gzip_size
entry-gutenboarding     +867 B  (+0.0%)        +33 B  (+0.0%)      +24 B  (+0.0%)

Total                   +867 B                 +33 B               +24 B
Chunks Delta:
name                 stat_size           parsed_size           gzip_size
entry-gutenboarding     +867 B  (+0.0%)        +33 B  (+0.0%)      +24 B  (+0.0%)

Total                   +867 B                 +33 B               +24 B
```

So no, I don't think we've been successful in avoiding it. However, it's not something we _should_ be pulling in. Likely pruning down some Calypso functionality into shakeable modules will let us drop it again.